nginx.conf

```
@@ -46,7 +46,16 @@ http {
     deny all;
 
     location / {
-      resolver 10.0.0.2;
+    # NGINX uses its own name resolution, so we'll grab the first resolver defined for the machine.
+      resolver <%=
+        lines = Array.new
+        File.open('/etc/resolv.conf').each { |line| 
+            lines << line.split(" ") if line.start_with?("nameserver") 
+        }
+
+        nameserver = lines.dig(0,1) if lines.length > 0
+        nameserver || "127.0.0.1"
```

```suggestion
        "169.254.0.2"
```
(github suggestions doesn't let you replace multiple lines, so this suggestion is the single-line change which communicates my intent)

I see what you're doing here and I think it's good to not hardcode the resolver to 10.0.0.2 (which is a magic value which happens to work on cloud.service.gov.uk). However, based on the discussion at cloudfoundry/nginx-buildpack#12, I think we should just hardcode the resolver to `169.254.0.2` for now, since that's the default for cloud foundry installations, and if and when we move to nginx-buildpack and cloudfoundry/nginx-buildpack#12 is merged, we should use `{{nameserver}}`.

---

now it's `{{nameservers}}`